app/views/govuk_publishing_components/components/_password_input.html.erb

```diff
@@ -0,0 +1,80 @@
+<%
+  add_gem_component_stylesheet("password-input")
+
+  label_text ||= t("components.password_input.label")
+
+  # This overrides the 'Show/Hide' button text, but this button text is getting overwritten by GOVUK Frontend JS currently when you interact with the button. Therefore this functionality is not being exposed in the docs.
+  button_text ||= t("components.password_input.button")
+  button_aria_label_text ||= t("components.password_input.button_aria_label")
+
+  error_text ||= nil
+  error_text_prefix ||= t("components.password_input.error_text_prefix")
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
```

Comment:
Nice. Also is it worth adding the shared helper for margin? It's usually the first thing we need.

Side note: I'm planning on moving the margin functionality into the component wrapper helper at some point, but it's a non-trivial task.